File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/ReviewResponse.java
package com.cinema.ticket_booking.data.model.response;

import com.google.gson.annotations.SerializedName;

public class ReviewResponse {
    @SerializedName("id")
    private String id;
    @SerializedName("userFullName")
    private String userFullName;
    @SerializedName("userAvatarUrl")
    private String userAvatarUrl;
    @SerializedName("rating")
    private int rating;
    @SerializedName("comment")
    private String comment;
    @SerializedName("createdAt")
    private String createdAt;

    public String getId() {
        return id;
    }

    public String getUserFullName() {
        return userFullName;
    }

    public String getUserAvatarUrl() {
        return userAvatarUrl;
    }

    public int getRating() {
        return rating;
    }

    public String getComment() {
        return comment;
    }

    public String getCreatedAt() {
        return createdAt;
    }
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/ReviewResponse.kt
package com.cinema.ticket_booking.data.model.response

import com.google.gson.annotations.SerializedName

data class ReviewResponse(
    @SerializedName("id")
    val id: String?,

    @SerializedName("userFullName")
    val userFullName: String?,

    @SerializedName("userAvatarUrl")
    val userAvatarUrl: String?,

    @SerializedName("rating")
    val rating: Int,

    @SerializedName("comment")
    val comment: String?,

    @SerializedName("createdAt")
    val createdAt: String?
)
